XCTest.xcodeproj/project.pbxproj

```diff
@@ -409,13 +407,22 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				MACOSX_DEPLOYMENT_TARGET = 10.9;
+				INFOPLIST_FILE = Info.plist;
+				IPHONEOS_DEPLOYMENT_TARGET = 10.0;
```

Comment:
There're two reasons, see them below:

1. `swift-corelibs-xctest` uses [Timer.scheduledTimer(withTimeInterval:repeats:block:)](https://github.com/apple/swift-corelibs-xctest/blob/242864abf579529281ed29932c58341fc8962525/Sources/XCTest/Private/XCPredicateExpectation.swift#L36) that requires iOS 10.0+, macOS 10.12+, tvOS 10.0+, watchOS 3.0+. The [link](https://developer.apple.com/reference/foundation/timer/2091889-scheduledtimer) to the documentation.
2. `SwiftFoundation` macOS deployment target is 10.11.

PS. I think that at the moment we don't need deployment targets lower that specified. We can decrease deployment target as soon as people need it, but in this case we need to make some changes in the code.